tests/foreman/destructive/test_auth.py

```diff
@@ -42,7 +42,7 @@ def test_positive_password_reset(target_sat):
     """
     result = target_sat.execute('foreman-rake permissions:reset')
     assert result.status == 0
-    reset_password = result.stdout.strip().split('password: ')[1]
+    reset_password = result.stdout.strip().split('\n')[0].split('password: ')[1]
```

Comment:
Just to simplify this :)
```suggestion
    reset_password = result.stdout.splitlines()[0].split('password: ')[1]
```

Reply:
Or `reset_password = result.stdout.splitlines()[0].split()[-1]`